[2024-12-30T22:48:47.579+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-30T22:48:47.593+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_kafka_producer.generate_and_send_email scheduled__2024-12-30T22:48:40+00:00 [queued]>
[2024-12-30T22:48:47.600+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_kafka_producer.generate_and_send_email scheduled__2024-12-30T22:48:40+00:00 [queued]>
[2024-12-30T22:48:47.601+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-30T22:48:47.615+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): generate_and_send_email> on 2024-12-30 22:48:40+00:00
[2024-12-30T22:48:47.622+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1519) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-30T22:48:47.624+0000] {standard_task_runner.py:72} INFO - Started process 1520 to run task
[2024-12-30T22:48:47.624+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_kafka_producer', 'generate_and_send_email', 'scheduled__2024-12-30T22:48:40+00:00', '--job-id', '1572', '--raw', '--subdir', 'DAGS_FOLDER/weather_kafka_producer.py', '--cfg-path', '/tmp/tmpiopg5i0v']
[2024-12-30T22:48:47.625+0000] {standard_task_runner.py:105} INFO - Job 1572: Subtask generate_and_send_email
[2024-12-30T22:48:47.664+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_kafka_producer.generate_and_send_email scheduled__2024-12-30T22:48:40+00:00 [running]> on host bb8001553f86
[2024-12-30T22:48:47.738+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_kafka_producer' AIRFLOW_CTX_TASK_ID='generate_and_send_email' AIRFLOW_CTX_EXECUTION_DATE='2024-12-30T22:48:40+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-30T22:48:40+00:00'
[2024-12-30T22:48:47.739+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-30T22:48:47.763+0000] {baseoperator.py:416} WARNING - EmailOperator.execute cannot be called outside TaskInstance!
[2024-12-30T22:48:47.771+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/email.py:155: RemovedInAirflow3Warning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
  send_mime_email(e_from=mail_from, e_to=recipients, mime_msg=msg, conn_id=conn_id, dryrun=dryrun)

[2024-12-30T22:48:47.772+0000] {configuration.py:1073} WARNING - section/key [smtp/smtp_user] not found in config
[2024-12-30T22:48:47.772+0000] {email.py:272} INFO - Email alerting: attempt 1
[2024-12-30T22:48:47.876+0000] {email.py:283} INFO - Sent an alert email to ['your_email@example.com']
[2024-12-30T22:48:47.881+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-30T22:48:47.892+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-30T22:48:47.893+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_kafka_producer, task_id=generate_and_send_email, run_id=scheduled__2024-12-30T22:48:40+00:00, execution_date=20241230T224840, start_date=20241230T224847, end_date=20241230T224847
[2024-12-30T22:48:47.920+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-30T22:48:47.936+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-30T22:48:47.938+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
